index.bs

```diff
@@ -954,6 +975,34 @@ these steps.
 :: Returns true if the user agent supports [=WebTransport sessions=] over exclusively reliable
    [=connections=], otherwise false.
 
+: <dfn for="WebTransport" attribute>anticipatedConcurrentIncomingUnidirectionalStreams</dfn>
+:: Optionally lets an application specify the number of concurrently open
+   [=incoming unidirectional=] streams it anticipates the server creating.
+   If not null, the user agent SHOULD attempt to reduce future round-trips by taking
+   {{[[AnticipatedConcurrentIncomingUniStreams]]}} into consideration in its
+   [=implementation-defined=] calculation of cumulative values for MAX_STREAM frames
+   it sends to the server.
+   [[!QUIC]][Section 19.11](https://www.rfc-editor.org/rfc/rfc9000#section-19.11)
+
+   The getter steps are to return [=this=]'s {{[[AnticipatedConcurrentIncomingUniStreams]]}}.
+
+   The setter steps, given |value|, are to set [=this=]'s
+   {{[[AnticipatedConcurrentIncomingUniStreams]]}} to |value|.
+
+: <dfn for="WebTransport" attribute>anticipatedConcurrentIncomingBidirectionalStreams</dfn>
+:: Optionally lets an application specify the number of concurrently open
+   [=bidirectional=] streams it anticipates the server creating.
+   If not null, the user agent SHOULD attempt to reduce future round-trips by taking
+   {{[[AnticipatedConcurrentIncomingBiStreams]]}} into consideration in its
+   [=implementation-defined=] calculation of cumulative values for MAX_STREAM frames
+   it sends to the server.
+   [[!QUIC]][Section 19.11](https://www.rfc-editor.org/rfc/rfc9000#section-19.11)
+
+   The getter steps are to return [=this=]'s {{[[AnticipatedConcurrentIncomingBiStreams]]}}.
+
+   The setter steps, given |value|, are to set [=this=]'s
+   {{[[AnticipatedConcurrentIncomingBiStreams]]}} to |value|.
```

Comment:
Do we need to add some language to indicate that the user-agent is not guaranteed to provide the number of streams requested by the API?

Reply:
Isn't that captured by the SHOULD?